tests/cypress/integration/actions_tasks_objects/case_15_group_features.js

```diff
@@ -0,0 +1,132 @@
+        points: 'By 2 Points',
+        type: 'Track',
+        switchLabel: false,
+        firstX: createRectangleTrack2Points.firstX + 300,
+        firstY: createRectangleTrack2Points.firstY,
+        secondX: createRectangleTrack2Points.secondX + 300,
+        secondY: createRectangleTrack2Points.secondY,
+    };
+
+    let fillChangeAppearance = '';
+    let fillGroupedShape = '';
+    let fillGroupedTrack = '';
+
+    before(() => {
+        cy.openTaskJob(taskName);
+    });
+
+    describe(`Testing case "${caseId}"`, () => {
+        it('Create two shapes and two tracks.', () => {
+            cy.createRectangle(createRectangleShape2Points);
+            cy.createRectangle(createRectangleShape2PointsSecond);
+            cy.createRectangle(createRectangleTrack2Points);
+            cy.createRectangle(createRectangleTrack2PointsSecond);
+        });
+        it('Set option "Color by" to "Group".', () => {
+            cy.changeAppearance('Group');
+            cy.get('.cvat_canvas_shape').then($listCanvasShapes => {
+                for (let i=0; i<$listCanvasShapes.length; i++) {
+                    cy.get($listCanvasShapes[i]).should('have.css', 'fill').then($fill => {
+                        // expected rgb(224, 224, 224) to equal rgb(224, 224, 224)
+                        expect($fill).to.be.equal($fill);
+                        fillChangeAppearance = $fill;
+                    });
+                }
+            });
+            cy.get('.cvat-objects-sidebar-state-item').then($listObjectsSidebarStateItem => {
+                for (let i=0; i<$listObjectsSidebarStateItem.length; i++) {
+                    cy.get($listObjectsSidebarStateItem[i]).should('have.css', 'background-color').then($bColorObjectsSidebarStateItem => {
+                        // expected rgba(224, 224, 224, 0.533) to include [ 224, 224, 224, index: 4, input: 'rgb(224, 224, 224)', groups: undefined ]
+                        expect($bColorObjectsSidebarStateItem).contain(fillChangeAppearance.match(/\d+, \d+, \d+/));
+                    });
+                }
+            });
+        });
+        it('With group button unite two shapes. They have corresponding colors.', () => {
+            cy.get('.cvat-group-control').click();
+            for (let shapeToGroup of ['#cvat_canvas_shape_1', '#cvat_canvas_shape_2']) {
+                cy.get(shapeToGroup).click();
+            }
+            cy.get('.cvat-group-control').click();
+            for (let groupedShape of ['#cvat_canvas_shape_1', '#cvat_canvas_shape_2']) {
+                cy.get(groupedShape).should('have.css', 'fill').then($fillGroupedShape => {
+                    // expected rgb(250, 50, 83) to not equal rgb(224, 224, 224)
+                    expect($fillGroupedShape).to.not.equal(fillChangeAppearance);
+                    // expected rgb(250, 50, 83) to equal rgb(250, 50, 83)
+                    expect($fillGroupedShape).to.be.equal($fillGroupedShape);
```

Comment:
``expect($fillGroupedShape).to.be.equal($fillGroupedShape);``?

Reply:
As well as for ```$fill```